README.md

```diff
@@ -41,8 +41,12 @@ Sample use
               CLIENT_ID: ${{ secrets.NON_PROD_SPN_ID }}
               CLIENT_SECRET: ${{ secrets.NON_PROD_SPN_SECRET }}
 
+Latest add-on 
```

Comment:
Instead of doing this, let's modify the example in the readme to use the options you filled out